src/Reporter/AbstractBaseReporter.php

```diff
@@ -102,7 +102,13 @@ public function __construct(
      */
     public function color($key, $text)
     {
-        if (!$this->configuration->areColorsEnabled() || !$this->hasColorSupport()) {
```

Comment:
This reads a little awkwardly to me. Maybe we can extract these conditions out into variables?

```php
$colorsEnabled = $this->configuration->areColorsEnabled() && $this->hasColorSupport();
$colorsExplicitlyEnabled = $this->configuration->areColorsEnabledExplicit();

if (!$colorsEnabled && !$colorsExplicitlyEnabled) {
    return $text;
}
```

thoughts?